INFO: Starting fuzzing
WARNING: No test specified, defaulting to first test defined (_)
input starts
EXTERNAL: qs v0.23.5.

nfolds values: -2.43318e-187
nrow values: -1.64581e+174
input ends

error: Col::subvec(): indices out of bounds or incorrectly used
terminate called after throwing an instance of 'std::logic_error'
  what():  Col::subvec(): indices out of bounds or incorrectly used
ERROR: Failed: _
INFO: Saved test case in file `/home/akhila/fuzzer_packages/fuzzedpackages/midasml/inst/testfiles/cvfolds/cvfolds_output/edf1d2f95c1b751784ef04e63a9c60a4d95ab540.fail`
input starts
EXTERNAL: qs v0.23.5.

nfolds values: -1.5615e-299
nrow values: -1.20777e+13
input ends

error: Col::subvec(): indices out of bounds or incorrectly used
terminate called after throwing an instance of 'std::logic_error'
  what():  Col::subvec(): indices out of bounds or incorrectly used
ERROR: Failed: _
INFO: Saved test case in file `/home/akhila/fuzzer_packages/fuzzedpackages/midasml/inst/testfiles/cvfolds/cvfolds_output/2933719dc74e96a12995827905fce0809c41dde9.fail`
input starts
EXTERNAL: qs v0.23.5.

nfolds values: 1.27048e-62
nrow values: -2.88162e-134
input ends

error: Col::subvec(): indices out of bounds or incorrectly used
terminate called after throwing an instance of 'std::logic_error'
  what():  Col::subvec(): indices out of bounds or incorrectly used
ERROR: Failed: _
INFO: Saved test case in file `/home/akhila/fuzzer_packages/fuzzedpackages/midasml/inst/testfiles/cvfolds/cvfolds_output/928a5b2346d40f728026e970e7974acdc472fa54.fail`
input starts
EXTERNAL: qs v0.23.5.

nfolds values: 3.16522e+06
nrow values: 3.72126e-82
input ends

error: Col::subvec(): indices out of bounds or incorrectly used
terminate called after throwing an instance of 'std::logic_error'
  what():  Col::subvec(): indices out of bounds or incorrectly used
ERROR: Failed: _
INFO: Saved test case in file `/home/akhila/fuzzer_packages/fuzzedpackages/midasml/inst/testfiles/cvfolds/cvfolds_output/9be2e84527303d576bf8fbd05c45d5e60d9e5d37.fail`
INFO: Done fuzzing! Ran 4 tests (2 tests/second) with 4 failed/0 passed/0 abandoned tests
